src/parser.ts

```diff
@@ -9102,7 +9102,7 @@ export function parseJSXClosingFragment(
   if (inJSXChild) {
     consume(parser, context, Token.GreaterThan);
   } else {
-    consume(parser, context, Token.GreaterThan);
+    scanJSXToken(parser, context);
   }
```

Comment:
Another existing bug due to copy paste I assume. Note the previous two branches are same.